test/test_helpers.js

```diff
@@ -37,9 +37,9 @@ function getExtension(str) {
     // So, the result we want is the third Array element,
     // since the first one is the whole match, the second one
     // returns the first captured match, etc.
-    const re = /(\.)([a-zA-Z0-9]+)$/;
+    const match = str.match(/(\.)([a-zA-Z0-9]+)$/);
 
-    return str.match(re)[2];
+    return match && match[2];
```

Comment:
Will this return the value of match[2]? I’d assume it would return a boolean.

Reply:
You'd think so but it's not :)

The use of `&&` there takes care of it.

Example from when running the endpoints:

```
null
null
null
null
json
null
null
null
null
txt
null
xml
```

And for the functional tests, it just returns the extension as usual, otherwise tests would fail there.

Reply:
So basically it's like saying, if match isn't null then return match[2].